awx/main/utils/ansible.py

```diff
@@ -106,7 +106,7 @@ def could_be_inventory(project_path, dir_path, filename):
 def read_ansible_config(project_path, variables_of_interest):
     fnames = ['/etc/ansible/ansible.cfg']
     if project_path:
-        fnames.insert(0, os.path.join(project_path, 'ansible.cfg'))
+        fnames.insert(1, os.path.join(project_path, 'ansible.cfg'))
```

Comment:
Would we just want `fnames.append` ?

Reply:
I wasn't sure if it was done this way for another reason so I fixed it with as little changing as possible.

Reply:
Yea, I'm just saying that based on what you're doing here, `names.append(os.path.join(project_path, 'ansible.cfg'))` is more succinct and accomplishes the same thing.